src/oop/oop1.py

```diff
@@ -17,3 +17,43 @@
 #     pass
 #
 # Put a comment noting which class is the base class
+class Vehicle:
+    '''
+    this is the Vehicle base (parent) class
+    '''
+
+class FlightVehicle(Vehicle):
+    '''
+    this is the FlightVehicle class, subclass of Vehicle
+    '''
+    pass
+
+class Starship(FlightVehicle):
+    '''
+    this is the Starship class, subclass of FlightVehicle
+    '''
+    pass
+
+class Airplane(FlightVehicle):
+    '''
+    this is the Airplane class, subclass of Flightvehicle
+    '''
+    pass
+
+class GroundVehicle(Vehicle):
+    '''
+    this is the GroundVehicle class, subclass of Vehicle
+    '''
+    pass
+
+class Car(GroundVehicle):
+    '''
+    this is the Car class, subclass of GroundVehicle
+    '''
+    pass
+
+class Motorcycle(GroundVehicle):
```

Comment:
Nice work, @taniamichelle ! You got all the tests to pass! 
<img width="778" alt="Screen Shot 2019-12-09 at 2 40 21 AM" src="https://user-images.githubusercontent.com/16110931/70416557-54655900-1a2d-11ea-99d3-b42a3c8aea92.png">